src/components/periodic-report/handlers/sync-progress-report-to-engagement.handler.ts

```diff
@@ -40,6 +40,10 @@ export class SyncProgressReportToEngagementDateRange
       event: event.constructor.name,
     });
 
+    if (event instanceof EngagementCreatedEvent && event.engagement.changeset) {
+      return;
+    }
```

Comment:
When I changeset is approved is an engagement updated event published to trigger progress reports to be created for the new engagement?

Reply:
I skip processing progress reports in changeset mode for both Created/Updated events.